src/bento/dashboardTabData.js

```diff
@@ -75,7 +75,7 @@ export const tabIndex = [
 ];
 
 export const DASHBOARD_QUERY_NEW = gql`
-query search (
+query search1 (
```

Comment:
@huuaho Is it possible to call query-only "search"? Is there any specific reason we need to have search1, Search2, and so on..

Reply:
it was for debugging, and if i wanted to figure out which "search" query was throwing an error, the numbers are now removed.

Reply:
@huuaho I can not see any changes or any new commit.

Reply:
apologies, did not push to repo